docs/01-programs/02-candy-machine/10-how-to-guides/00-my-first-candy-machine.md

```diff
@@ -0,0 +1,600 @@
+
+:::info
+Solana is designed to be a high-throughput, low-latency blockchain that can host smart contracts, but one of the design tradeoffs is that storing data on Solana is fairly expensive. For NFTs, we only store basic information about the NFT, such as the name, symbol, creators, and seller fee basis points. For the image and all the other metadata, we have a URL on-chain on Solana that points to an external JSON file containing all that information.
+
+This URL can point to any JSON file anywhere. It can be a decentralized file storage blockchain like Arweave or Filecoin, a peer-to-peer distributed file system like IPFS, or even a centralized cloud server like AWS.
+
+Sugar currently supports four upload methods: Arweave via Bundlr, IPFS via NFT Storage or Pinata, Shadow Drive and AWS.
+:::
+
+In our previous step we selected `Bundlr` as our upload method, and we saw this in the config file when we viewed it. Now, all we have to do is run the following command:
+
+```
+sugar upload
+```
+
+<details>
+<summary>Output</summary>
+<p>
+
+```
+[1/4] 🗂 Loading assets
+Found 10 asset pair(s), uploading files:
++--------------------+
+| images | 10 |
+| metadata | 10 |
++--------------------+
+
+[2/4] 🖥 Initializing upload
+▪▪▪▪▪ Connected
+Funding address:
+-> pubkey: DF2eiJojmgvcXDoQgSFzhrnt9Ds4GoYZmaCRMF8sWMYN
+-> lamports: 5939049 (◎ 0.005939049)
+Signature: 5zeJw5WerTFVDTWEoCGDH6WwRcdFcsjAswKvZ6QB6vzSVtaWKK4QVRH3BsKHjHeCASXiN7W8ni7BsYxFuMAmjVUD
+
+[3/4] 📤 Uploading image files
+
+Sending data: (Ctrl+C to abort)
+[00:00:09] Upload successful ██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████ 10/10
+
+[4/4] 📤 Uploading metadata files
+
+Sending data: (Ctrl+C to abort)
+[00:00:00] Upload successful ██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████ 10/10
+
+10/10 asset pair(s) uploaded.
+
+✅ Command successful.
+```
+
+</p>
+</details>
+
+:::tip
+Sugar commands are designed to be robust and commands such as `upload` and `deploy` will pick up where they left off when you run them again. You may occasionally get an error such as:
+
+🛑 Error running command (re-run needed): Blockhash not found
+
+In such cases it's safe to rerun the command until it succeeds.
+:::
+
+:::caution
+However, some upload methods such as Bundlr, do cost funds to upload and store the data. If you successfully upload your data and then run it again, it will charge you again. If you do this repeatedly with a large amount of data it can cost you a significant amount of SOL. Once you have successfully uploaded your data you should not have to do it again, as the cache file will store all the links to the data. Ask on the Metaplex Discord if you run into any unexpected issues.
+:::
```

Comment:
Way too many callouts here. Especially if you include the "info" callout about and all the code accordions.

It feels like entering a very busy boulevard where all the shops are trying to get your attention. It's too distracting.

Please reword some of these sections so they feel like legitimate parts of this guide.

You're allowed to keep one callout if you must but the rule of thumb should be no more than one callout per `<h2>` section.